network/suite.go

```diff
@@ -63,3 +77,72 @@ func (s *TestSuite) CreateTxBytes(fees sdk.Coin, gas uint64, msgs []sdk.Msg) ([]
 	bz, err := val.ClientCtx.TxConfig.TxEncoder()(builder.GetTx())
 	return bz, err
 }
+
+func (s *TestSuite) GetCometClient() (*cmthttp.HTTP, error) {
+	return cmthttp.New(s.Network.Validators[0].RPCAddress, "/websocket")
+}
+
+// CreateTxBytes creates and signs a transaction, from the given messages.
+func (s *TestSuite) CreateTxBytes(ctx context.Context, acc account.Account, gasLimit, timeoutHeight uint64, fee sdk.Coins, chainID string, msgs ...sdk.Msg) ([]byte, error) {
```

Comment:
What happens if we need to modify the sequence number? I.e I'm submitting a bundle and need to set an incremented sequence?

Reply:
Maybe we add options?

`WithSequenceOverride()` ?

Reply:
nit: get chainID from the TestSuite?

Reply:
Ya, I was thinking j adding the sequence # as argument would suffice, either way works?